src/App.js

```diff
@@ -1,14 +1,17 @@
-import style from "./App.module.css";
 import { Header } from "./components/Header/Header";
+import { Map } from "./components/Main/Map";
+import style from "./App.module.css";
 
-function App() {
+export function App() {
   return (
     <div className={style.container}>
-      <Header className={style.header} />
+      <header className={style.header}>
+        <Header />
+      </header>
       <sidebar className={style.sidebar}>Sidebar</sidebar>
```

Comment:
`sidebar` is not a valid semantic HTML element. It works because React converts anything that it doesn't recognise to `div`.

Reply:
good to know, so use `aside` instead?